.circleci/config.yml

```diff
@@ -67,6 +69,15 @@ jobs:
           command: |
             cd languages/java/secure-memory
             ./scripts/test.sh
+      - run:
```

Comment:
The save cache step should be moved up. Ideally, save the cache as soon as we run the tests. So that all the dependencies are pulled in.